frontend/components/text-classifier/labeling-rules/RulesManagement.vue

```diff
@@ -1,46 +1,60 @@
 <template>
   <div v-if="isVisible" class="rules-management">
     <ReLoading v-if="$fetchState.pending" />
-    <div v-else-if="!$fetchState.error" class="rules-management__container">
-      <re-button
-        class="rules-management__close button-quaternary"
-        @click="hideList()"
-      >
-        <svgicon name="chev-left" width="12" height="12"></svgicon>Back to query
-        view</re-button
-      >
-      <p class="rules-management__title">Overall Metrics</p>
-      <rules-overall-metrics :rules="rules" :dataset="dataset" />
-      <p class="rules-management__title">Rules</p>
-      <ReSearchBar
-        v-if="formattedRules.length"
-        placeholder="Search rule by name"
-        @input="onSearch"
-      />
-      <ReTableInfo
-        class="rules-management__table"
-        :data="formattedRules"
-        :sorted-order="sortedOrder"
-        :sorted-by-field="sortedByField"
-        :columns="tableColumns"
-        :actions="actions"
-        :query-search="querySearch"
-        :global-actions="false"
-        search-on="name"
-        :no-data-info="noDataInfo"
-        :empty-search-info="emptySearchInfo"
-        :visible-modal-id="visibleModalId"
-        :delete-modal-content="getDeleteModalText"
-        @sort-column="onSortColumns"
-        @onActionClicked="onActionClicked"
-        @close-modal="closeModal"
-      />
-      <re-button
-        v-if="formattedRules.length"
-        class="button-primary"
-        @click="updateSummary()"
-        >Update Summary</re-button
+    <div v-else-if="!$fetchState.error">
```

Comment:
What happens if something went wrong? Is there any error handing defined?

Reply:
At view level we do not have anything that refers to the error, it could be contemplated. Right now, I think it shows the toast with the error and the component will be shown empty